packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -328,7 +328,23 @@ impl DatabaseApi<'_> {
         target_entity_id: EntityId,
         link_type_id: VersionedUri,
     ) -> Result<(), InsertionError> {
-        let link = Link::new(source_entity_id, target_entity_id, link_type_id);
+        let link = Link::new(source_entity_id, target_entity_id, link_type_id, None);
+        self.store.create_link(&link, self.account_id).await
+    }
+
+    async fn create_indexed_link(
```

Comment:
Nit: doesn't need to change, but I think would still prefer `ordered` over `indexed` to describe whether a link is ordered given that we have a `ordered` field on the entity type schemas to indicate whether an outgoing link has an `index`. But either works.

```suggestion
    async fn create_ordered_link(
```